examples/basic_usage.js

```diff
@@ -34,6 +34,7 @@ console.log();
 console.log('mixed use of data types');
 print(math.add(4, [5, 6]));                   // number + Array, [9, 10]
 print(math.multiply(math.unit('5 mm'), 3));   // Unit * number,  15 mm
+print(math.multiply(math.unit('8 ft'), math.unit('20 lbf')).to('ft lbf'));   // Unit * number,  15 mm
```

Comment:
It's indeed nice to showcase as much as possible in this very small example, but I would prefer to keep this example as it is, short and simple, and go nuts in the units.js example.